plugins/modules/net_tools/ldap/ldap_attr.py

```diff
@@ -35,6 +35,9 @@
     rules. This should work out in most cases, but it is theoretically
     possible to see spurious changes when target and actual values are
     semantically identical but lexically distinct.
+  - "The C(params) parameter was removed due to circumventing Ansible's parameter
```

Comment:
```suggestion
  - "The I(params) parameter was removed due to circumventing Ansible's parameter
```
(same in the next line)

Reply:
Thanks, good catch. I should have checked the formatting too.

Reply:
I overlooked it the first time as well...